src/components/ux-pricingtable/ux-pricingtable.feature

```diff
@@ -3,6 +3,14 @@ Feature: ux-pricingtable test suite
 	Scenario: Loading ux-pricingtable
 		Given I have loaded component "ux-pricingtable" with use case "dataDriven"
 		Then the element "pricing_table" will have the class "table"
+		And there will be an element for "title"
+		And there will be an element for "price"
+		And there will be an element for "description"
+		And there will be an element for "button"
+		And there are 3 "bullet" elements displayed
+		And the element "title" will have the text "Standard"
```

Comment:
Whoops, named the test step incorrectly. It should use the correct test from steps.js now.